### PR TITLE
Translate Support of Ruby 2.1 has ended (zh_tw)

### PR DESCRIPTION
**Pull request by bugtender**
zh_tw Translation for Support of Ruby 2.1 has ended

- zh_tw/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md

Thank you.

### CODE REVIEW

File: zh_tw/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+目前處於正常維護階段。如有需要，我們將會移植錯誤修正並發佈修正版本。如果發現嚴重安全性問題，我們會為此發佈緊急修正版本。
+
+### Ruby 2.3 系列
+
+目前處於正常維護階段。如有需要，我們將會移植錯誤修正並發佈修正版本。如果發現嚴重安全性問題，我們會為此發佈緊急修正版本。
+
+### Ruby 2.2 系列
+
+目前處於安全性維護階段，除了安全性修正外，我們將不會移植任何 錯誤修正回去 2.2 系列。如果發現嚴重安全性問題，我們會為此發佈緊急修正版本。我們計畫在 2018 年 3 月底停止 Ruby 2.2 系列的支援。
```
Comment (hlcfan) [inline zh_tw/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md]: There's an extra space `植任何 錯誤`

**Comment by bugtender** [comment]
Fixed it ! Thanks @hlcfan . 😆

**Comment by hlcfan** [comment]
👍

**Comment by hlcfan** [comment]
@JuanitoFatas Can you please help push the latest 2 translations to PROD?

**Comment by stomar** [comment]
done.

**Pull request merged**